commands/box/contract/root.go

```diff
@@ -1009,6 +1014,39 @@ func reset(cmd *cobra.Command, args []string) {
 	}
 }
 
+func contractdetails(cmd *cobra.Command, args []string) {
+	if len(args) > 1 {
+		fmt.Println(cmd.Use)
+		return
+	}
+	abi := new(abi.ABI)
```

Comment:
var abi abi.ABI
check whether index is set before
check file exists first, then parse argument to index